pom.xml

```diff
@@ -180,6 +180,13 @@
                 <version>${flink.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <!--Because the license is not in compliance, if you need to use MySQL, you can add it yourself-->
```

Comment:
```suggestion
            <!--Because the license is not in ASF compliance, if you need to use MySQL, you can add it yourself-->
```

Reply:
Maybe we should add keyword `ASF` or `Apache-2.0` and something else here?

Reply:
Our license is apache 2.0, maybe there is no need to declare again here

Reply:
You're right 👍